PROV/prov-cco-directmappings.ttl

```diff
@@ -51,41 +52,41 @@ prov:wasAssociatedWith    rdfs:subPropertyOf cco:has_agent ;
 prov:wasInvalidatedBy     rdfs:subPropertyOf cco:is_affected_by ;
                           rdfs:comment "An Entity that was invalidated by an Activity is also affected by that Activity."@en .
 prov:wasGeneratedBy       rdfs:subPropertyOf cco:is_output_of ;
-                          rdfs:comment "An Entity that was generated by an Activity is output of that Activity"@en .
+                          rdfs:comment "An Entity that was generated by an Activity is output of that Activity."@en .
 prov:wasInformedBy        rdfs:subPropertyOf cco:caused_by ;
-                          rdfs:comment "If the Activity A1 was informed by the Activity A2, then A1 is caused by A2"@en .
+                          rdfs:comment "If the Activity A1 was informed by the Activity A2, then A1 is caused by A2."@en .
 
 prov:invalidated          rdfs:subPropertyOf cco:affects ;
                           rdfs:comment "An Activity that invalidated an Entity also affects that Entity."@en .
 prov:used                 rdfs:subPropertyOf cco:affects ;
                           rdfs:comment "An Activity that used an Entity affects the Entity."@en .
 prov:generated            rdfs:subPropertyOf cco:has_output ;
-                          rdfs:comment "An Activity that generated an Entity has_output that Entity"@en .
+                          rdfs:comment "An Activity that generated an Entity has_output that Entity."@en .
 
 
 # "The _optional_ Plan adopted by an Agent in Association with some Activity. Plan specifications are out of the scope of this specification."
 prov:hadPlan                  rdfs:subPropertyOf cco:has_input ; # an Influence has_input some Plan
```

Comment:
Second that, it seems that it is not neccessary to specify a plan at the very start of an activity.

Reply:
The domain here is prov:Association, so it might be related to some kind of reification. From the definition of Association:
> An activity association is an assignment of responsibility to an agent for an activity

I don't think we can say the plan necessarily prescribed the association or was even "about" the association... I think plans in these contexts are usually about activities. Here's an example
```turtle
:illustrating
    a prov:Activity; 
    prov:wasAssociatedWith :derek, 
                           :steve;
    prov:qualifiedAssociation [
        a prov:Association;
        prov:agent   :derek;
        prov:hadRole :illustrationist;
    ];
    prov:qualifiedAssociation [
        a prov:Association;
        prov:agent   :steve;
        prov:hadRole :stylist;
        prov:hadPlan :style-guide;
        rdfs:comment "Steve helped Derek conform with the publisher's style guide."@en;
    ];
.
```
I'll try a property chain like this:
```
[Activity] cco:prescribed_by [Directive Information Content] o [Plan] inverse(hadPlan) [Association]
```

Reply:
Oops I mean this: `inverse(qualifiedAssociation) o prescribed_by`